### ISSUE TITLE
Add HODL type of investment

### ISSUE THREAD

**Issue by AndreaBarghigiani** [AndreaBarghigiani/profit-tracker#3]
Allow a user to open a type of investment called HODL, this will be useful for the kind of DCA or speculative operations an investor can make. The base theory is that I buy low an asset and I'll sell it for profit.

This is the user story:
- click on *"Create HODL investment"*
- the page will ask for `token`, `amount` and `currencyAmount` (this will help to track conversion like *"1 ETH used to buy BTC"*)
- the app will check the `tokens` table to see if they'll already tracked
  - if both are tracked, we will go ahead and save the deposit
  - if not tracket, (for now) we ask the user to require token addition to the list
- once deposit is complete, user can track the value of his investment in the dashboard
- inside the investment page, user will be able to add new `deposit`s as well as record `sell`s (those will be tracked as `withdraw` type of transactions)

We will keep an history of all tracked tokens via the CoinMarketCap API, a cron will be fired every 24h.
